src/Plugin.InAppBilling.iOS/InAppBillingImplementation.cs

```diff
@@ -56,15 +56,18 @@ public InAppBillingImplementation()
 		{
 			var products = await GetProductAsync(productIds);
 
-			return products.Select(p => new InAppBillingProduct
-			{
-				LocalizedPrice = p.LocalizedPrice(),
-				MicrosPrice = (long)(p.Price.DoubleValue * 1000000d),
-				Name = p.LocalizedTitle,
-				ProductId = p.ProductIdentifier,
-				Description = p.LocalizedDescription,
-				CurrencyCode = p.PriceLocale?.CurrencyCode ?? string.Empty
-			});
+            var operatingSystemHasIntroductoryPrice = NSProcessInfo.ProcessInfo.IsOperatingSystemAtLeastVersion(new NSOperatingSystemVersion(11, 2, 0));
```

Comment:
This is wrong. Put it at the class level:

bool IsiOS112 => UIDevice.CurrentDevice.CheckSystemVersion (11,2);